corehq/motech/openmrs/finders.py

```diff
@@ -54,11 +72,18 @@ class PatientFinder(DocumentSchema):
     """
 
     # Whether to create a new patient if no patients are found
-    create_missing = BooleanProperty(default=False)
+    create_missing = SchemaProperty(ValueSource, default=constant_false)
```

Comment:
This commit is to allow a project to configure their integration so that if a patient has been given an OpenMRS ID by a mobile worker, and that patient is not found, then it should NOT be created, because the patient does exist in OpenMRS, but their ID has been entered incorrectly by the mobile worker. (i.e. This is to avoid duplicate records.)

ValueSource subclasses fetch values from cases, form questions, or return constants.